video2dataset/dataloader/transform.py

```diff
@@ -130,7 +132,7 @@ def __call__(self, data):
                 frame = cv2.resize(
                     frame,
                     tuple(reversed(resize_size)),
-                    interpolation=cv2.INTER_LANCZOS4,
+                    interpolation=cv2.INTER_AREA,  # cv2.INTER_LANCZOS4,
```

Comment:
I think we want INTER_LANCZOS, why are you changing this? Its important for CLIP stuff

Reply:
yea sorry this was just for testing stuff to see if this was causing the bad magnitudes.